packages/blocks/src/page-block/edgeless/components/toolbar/brush/brush-tool-button.ts

```diff
@@ -88,9 +112,36 @@ export class EdgelessBrushToolButton extends LitElement {
     }
   }
 
+  override connectedCallback() {
+    super.connectedCallback();
+    const observer = new MutationObserver(() => {
+      // add mouse hover event to pen icon
+      this._penIcon.addEventListener(
+        'mouseenter',
+        this._handleMouseEnter.bind(this)
+      );
+      this._penIcon.addEventListener(
+        'mouseleave',
+        this._handleMouseLeave.bind(this)
+      );
+      observer.disconnect();
+    });
+
+    if (!this.shadowRoot) return;
+    observer.observe(this.shadowRoot, { childList: true });
```

Comment:
You can use `setTimeout` to wait for the DOM load

```js
      await setTimeout(0); // just example
      // add mouse hover event to pen icon
      this._penIcon.addEventListener(
        'mouseenter',
        this._handleMouseEnter.bind(this)
      );
      this._penIcon.addEventListener(
        'mouseleave',
        this._handleMouseLeave.bind(this)
      );
```

Reply:
Thanks, I will rewrite this part.